config/karma.js

```diff
@@ -0,0 +1,16 @@
+var grunt = require('grunt');
+var testOption = grunt.option('tests');
```

Comment:
this doesn't look it's used. Were you adding a way to run a single file? That would be nice. The way it works for fxpay is pretty useful: https://github.com/mozilla/fxpay#running-tests